.data/dashboard.example.json

```
@@ -70,6 +75,7 @@
         "size": [1, 1],
         "id": "88e12934-c628-48cb-8b84-b16c2e311105"
       }
-    ]
+    ],
```

I noticed that the example dashboard is missing a title value.

```suggestion
    ],
    "title": "⚡electric io",
```